src/notifications/notifications.service.ts

```
@@ -0,0 +1,117 @@
+    const result = await this.notificationRepository
+      .createQueryBuilder()
+      .update(Notification)
+      .set({ isRead: true })
+      .where('tenantId = :tenantId', { tenantId })
+      .andWhere('userId = :userId', { userId })
+      .andWhere('isRead = false')
+      .execute();
+
+    return { updated: result.affected ?? 0 };
+  }
+}
```

ESLint

[error] Newline required at end of file but not found.

(eol-last)